frontend/src/layout/header/UserBox.tsx

```diff
@@ -202,7 +202,10 @@ const LoggedIn: React.FC<LoggedInProps> = ({ user }) => {
             icon: <FiVideo />,
             wrapper: <ExternalLink
                 service="STUDIO"
-                params={{ "return.target": new URL(document.location.href) }}
+                params={{
+                    "return.target": new URL(document.location.href),
+                    "return.label": "Tobira",
```

Comment:
So mh, is "Tobira" the right string to pass there? Do you think the video portal name from the settings (which is shown as title of the root realm) is better? It could be quite long though, mhhh....

Reply:
Yeah, I think that might actually be better than just "Tobira". Might be long, sure, but hopefully institutions won't configure a name that is too long (I think that would also look a bit awkward on the homepage).

Reply:
> Might be long, sure, but hopefully institutions won't configure a name that is too long (I think that would also look a bit awkward on the homepage).

https://tobira.unibe.ch/
"Video Portal University of Bern (Tobira)"

![image](https://github.com/elan-ev/tobira/assets/7419664/ab75510a-f04a-4ea8-b496-54e1a5f53911)

^ @dagraf :P 

Ok seriously, I think Bern's title is still fine on the home page but might be a little bit crammed in the studio button. But oh well. Studio could also just limit the length of the button label. So yeah, let's do it like this.